src/main/api/2/studio-api-2.yaml

```diff
@@ -6330,6 +6330,197 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /workflow/request_publish:
+    post:
+      tags:
+        - workflow
+      summary: Request publish for content
+      description: 'Required permission "site_member"'
+      operationId: workflowRequestPublish
+      requestBody:
+        description: parameters for request publish
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties: 
+                items:
+                  type: array
+                  description: path(s) of content item(s)
+                  items:
+                    type: string
+                optional:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                mandatory:
+                  type: array
+                  description: mandatory (hard) dependencies
+                  items:
+                    type: string
+                environment:
+                  type: string
+                  description: environment to publish content to
+                schedule:
+                  type: string
+                  format: datetime
+                  description: schedule when to publish content
```

Comment:
Is this optional? Can it receive empty string or not be in the request at all when schedule is now or what should the value be when "now"?

Reply:
@dejan-brkic this can be omitted since it's optional and that means `now`, please confirm

Reply:
I don't want to mix string values with dates. empty means now

Reply:
Omitted = now